docker-compose/cassandra-3.11/README.md

```diff
@@ -31,25 +42,22 @@ Both scripts support the following options:
 
 * You can enable reguest logging using `-q`: if so, each request is logged under category `io.quarkus.http.access-log`
 
-# Notes
+## Notes
 
-* The `.env` file defines variables for the docker compose project name (`COMPOSE_PROJECT_NAME`),
-  and the Cassandra Docker image tag to use (`CASSTAG`).
+* The `.env` file defines variables for the docker compose project name (`COMPOSE_PROJECT_NAME`), the Cassandra Docker image tag to use (`CASSTAG`), and the Stargate Docker image tag to use (`SGTAG`).
 
-* The Stargate Docker image tag to use (`SGTAG`) defaults to the current version as defined in the
-  top level project `pom.xml` file. It can be overridden with the `-t` option on either script:
+* When using the convenience scripts, the Stargate Docker image (`SGTAG`) used for the `-l` option is the current (snapshot) version as defined in the top level project `pom.xml` file. It can be overridden with the `-t` option on either script:
 
-  `./start_cass_311.sh -t v2.0.0-BETA-2`
+  `./start_cass_311.sh -t v2.0.1`
```

Comment:
I would say we don't need the start scripts anymore? Ah OK, there are something there like request logs etc, but this would all be part of the `.env` now?

Reply:
They provide value as a convenience, especially the `-l` option.